### PR TITLE
Update cmake include directories

### PR DESCRIPTION
**Pull request by Jairard**
See https://github.com/Chlumsky/msdf-atlas-gen/issues/25.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -60,7 +59,7 @@ add_library(msdfgen::msdfgen ALIAS msdfgen)
 set_target_properties(msdfgen PROPERTIES PUBLIC_HEADER "${msdfgen_HEADERS}")
 target_include_directories(msdfgen INTERFACE
 	$<INSTALL_INTERFACE:include>
```
Comment (SpaceIm) [inline CMakeLists.txt:61]: this is also wrong if I understand that others projects should do `#include <msdfgen.h>`

File: CMakeLists.txt [hunk 2]
```diff
@@ -90,7 +89,7 @@ target_link_libraries(msdfgen-ext PUBLIC msdfgen::msdfgen Freetype::Freetype)
 target_include_directories(msdfgen-ext
 PUBLIC
 	$<INSTALL_INTERFACE:include>
```
Comment (SpaceIm) [inline CMakeLists.txt:91]: same comment than https://github.com/Chlumsky/msdfgen/pull/126/files#r685122553

File: CMakeLists.txt (hunk 1 above)
Comment (Jairard) [reply CMakeLists.txt:61]: @Chlumsky what do you think about this ? Do you prefer allowing both ways of including the lib or only allowing `<msdfgen.h>` and making a breaking update ?

Comment (Chlumsky) [reply CMakeLists.txt:61]: Why is it also wrong other than breaking backwards compatibility for the include?

**Pull request merged**